### DISCUSSION TITLE
Add Description for Custom Key Remap

### DISCUSSION THREAD

**Discussion by badjr13** [helix-editor/helix#7659] [Q&A] [answered]
Is it possible to add a description to my custom remap?

For example, I have this custom keymap in my config:

```toml
[keys.normal."space"]
"c" = ":buffer-close"
```

This allows me to hit `space` followed by `c` to close the current buffer, however when I hit space the space mode menu pops up and all of the default keymaps have their description but my custom keymap just has brackets. See attached image. Is it possible to replace the brackets with a custom description of my custom keymap?

![Screenshot from 2023-07-17 14-30-52](https://github.com/helix-editor/helix/assets/7246347/efbd0a69-ef4a-4143-859b-e144da82ee71)

**Comment by the-mikedavis** [comment, marked as answer]
Not currently, see https://github.com/helix-editor/helix/issues/1752

**Reply by badjr13** [reply]
Ah, thank you. I searched the repo for "description" instead of "label". Sorry, I didn't locate this one on my own.
